docs/unified-test-documentation/dasharo-performance/405-fan-control.md

```diff
@@ -0,0 +1,165 @@
+# Dasharo Performance: Fan Control
+
```

Comment:
@PrzemyslawBanasiak13 please add this new test suite to the `test-matrix` dedicated for ASUS KGPE D-16.